ProteGO/Common/DangerStatusManager/DangerStatusManager.swift

```diff
@@ -24,7 +48,25 @@ final class DangerStatusManager: DangerStatusManagerType {
         }
     }
 
-    deinit {
-        self.tweakBindings.forEach(DebugMenu.unbind)
+    func updateCurrentDangerStatus() {
```

Comment:
Co sądzisz zmianie nazwy na `getCurrentStatusAndDeviceIds`? Ja to rozumiem bardziej jako pobieranie niż updatowanie statusu. I moim zdaniem warto zawrzeć `DeviceIds` w nazwie.

Reply:
Ok, jestem za tym aby dodać tam `AndDeviceIds` ale nie jest to get, bo ta metoda nic nie zwraca, tylko właśnie aktualizuje Valeta i stan wewnętrzny tego managera

Reply:
`updateCurrentStatusAndDeviceIds` będzie ok

Reply:
Ok, może być